opentimelineio_contrib/adapters/tests/test_xges_adapter.py

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2019 Igalia S.L
+# Copyright 2019 Thibault Saunier <tsaunier@igalia.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "Apache License")
+#
+# You may obtain a copy of the Apache License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the Apache License with the above modification is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the Apache License for the specific
+# language governing permissions and limitations under the Apache License.
```

Comment:
This is a slightly different license text than used in the rest of OTIO. Are you able to use the exact same license as in the other files?

Reply:
Sorry about that, I don't know what happened here :-)

Reply:
I have a doubt, I am still allowed to use:

```
# Copyright (C) 2019 Igalia S.L
# Copyright 2019 Thibault Saunier <tsaunier@igalia.com>
```

right?

Also I copied the previous version from somewhere in the codebase (not sure where to be honnest), so I guess there is a place where it is not correct :-)

Reply:
Let me check about the copyright...